src/mistral_inference/lora.py

```diff
@@ -69,8 +69,9 @@ def ignore_missing_keys(m: nn.Module, incompatible_keys: NamedTuple) -> None:
         self.register_load_state_dict_post_hook(ignore_missing_keys)
 
     def forward(self, x: torch.Tensor) -> torch.Tensor:
-        lora = self.lora_B(self.lora_A(x))
-        result: torch.Tensor = self.linear(x) + lora * self.scaling
+        lora_A_result = self.lora_A(x)
```

Comment:
did you check if its working ?

Reply:
> did you check if its working ?

Yes

Reply:
Yes, I verified the changes and confirmed that the functionality is working as expected.